todo/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addTask, toggleCompleted, deleteTask } from './actions/index';
+
+import TodoForm from './components/TodoForm';
+import TodoList from './components/TodoList';
+
+import { AppContainer, ListContainer, HeaderContainer } from './styles/App';
+
+class App extends Component {
+  render() {
+    return (
+      <AppContainer>
+        <HeaderContainer></HeaderContainer>
+        <ListContainer>
+          <TodoForm addTask={this.props.addTask} />
+          <TodoList listArray={this.props.store.listArray} toggleCompleted={this.props.toggleCompleted} deleteTask={this.props.deleteTask} />
```

Comment:
You can pass your `toggleCompleted` and `deleteTask` in TodoList without needing to prop drill